src/coreclr/build-runtime.cmd

```diff
@@ -86,6 +86,7 @@ set __RestoreOptData=1
 set __CrossArch=
 set __PgoOptDataPath=
 set __CMakeArgs=
+set __Ninja=0
```

Comment:
I think I would suggest setting ninja as the default build target.

Reply:
For this PR, I'm planning on making Ninja opt-in. That way we can merge this in outside of an infra rollout. In a future infra rollout, we'll make Ninja the default.